### PR TITLE
(TC-140) Pretty print JSON when scanning

### PR DESCRIPTION
**Pull request by rick**
Make JSON output from scan be pretty-printed by default

 **todo**

- [x] pretty-print when marshaling when no consumerURL is set
- [x] verify that pretty-printing works
- [x] verify that reports are still just marshalled for storage (not pretty-printed)
- [x] :question: refactor marshalling method call point down one level?
- [x] minimal test(s)
- [ ] :ship:

cc @puppetlabs/lumogon

### CODE REVIEW

**Comment by rick** [comment]
Cleared my head funk I think. This is probably ready for :eyes: / merge.

**Comment by johnmccabe** [comment]
@rick looks good, just one comment can you update the README to remove the references to using `jq` to pretty print as thats no longer necessary - using it to explore the returned data is still grand.

**Comment by rick** [comment]
Thanks! Updated README.md.

File: README.md [hunk 1]
```diff
@@ -26,14 +26,6 @@ containers you have running and what we can learn about them. The output from a
 Lumogon scan will be a [JSON](https://en.wikipedia.org/wiki/JSON) listing of all the
 containers found and what Lumogon could learn about them.
 
-Since JSON can be hard to read as one big unstructured message, you might want
-to use a "pretty-printer" like [jq](https://stedolan.github.io/jq/) to print the
-data in a more readable format.
-
-``` shell
-docker run --rm  -v /var/run/docker.sock:/var/run/docker.sock puppet/lumogon scan | jq
```
Comment (johnmccabe) [inline README.md]: Probably want to keep the scan command in there :)

Comment (rick) [inline README.md]: It's the little things ;-)

Comment (johnmccabe) [inline README.md]: I'm happy with who I am :)

4d275c4cbade79e7831427793f9e8715_the-sexiest-am-fr-enduro-hardtail-thread-please-read-the-opening-pedantic-memes_300-274

**Comment by johnmccabe** [comment]
:shipit:

**Pull request merged**